### ISSUE TITLE
Following link from block comment leads to multiple 404s

### ISSUE THREAD

**Issue by Vaporbook** [emberjs/website#2598]
Would love to see this prioritized as it really inhibits Ember newbies like myself from learning about how the framework actually works.

Repro:

A doc comment on the Application namespace in ember 1.13 contains a URL http://emberjs.com/guides/understanding-ember/the-view-layer/#toc_event-delegation. Following the URL leads to a redirect to https://guides.emberjs.com/v2.5.0/understanding-ember/the-view-layer/, which returns a 404. That 404 page contains some copy and a call to follow a link to "Guides" at https://guides.emberjs.com/guides, which also returns a 404. That 404 has the same bad link to the "Guides" section.

User is lost in space. I'm okay with the original 404 as long as I can find my way back to the mothership somehow. But fixing the first 404 would be awesome, even if it meant taking me to a "related topics" page.

**Comment by locks** [comment]
Unfortunately we cannot update API documentation to rectify the link, as that would require a new release, so the first 404 is unavoidable. We have discussed redirection strategies, but it's either "show them a 404", or "do some error-prone logic to figure out the best page to redirect to". At least with the 404 it's a consistent experience ^_^;
As for the link in the 404 page itself, that was [already addressed](https://github.com/emberjs/guides/pull/1422), but we haven't deployed since the change was made.
The "Understanding Ember.js" has since been removed, with some of the guides merged into others, and most removed due to being outdated, like the view layer specifically.

Thank you for bringing this to our attention, we do want newcomers to have the best experience we can offer. I am closing this issue as I hope that the 404 page fix is sufficient to provide a way forward until we can address legacy aspects of the routes.
